package.json

```diff
@@ -80,4 +80,4 @@
   "ember-addon": {
     "configPath": "tests/dummy/config"
   }
-}
+}
```

Comment:
Can you revert this please, as npm will add this back the next time it gets modified.